doc/abrt-action-save-package-data.conf.txt

```diff
@@ -18,30 +18,47 @@ and component name to problem data.
 The configuration file consists of items in the format "Option = Value".
 A description of each item follows:
 
-OpenGPGCheck = 'yes' | 'no'::
-   When set to 'yes', 'abrt' will handle only crashes in executables which belong
-   to an installed package with valid GPG signature.
-   The default is 'yes'.
-   The files containing trusted GPG keys which are used to verify
-   the signatures are listed in the gpg_keys.conf configuration file.
-BlackList = 'packageName', 'packageName' ...::
-   'abrt' will ignore packages in this list and will not handle their crashes.
-BlackListedPaths = '`/path/to/ignore/*`', '`*/another/ignored/path*`' ...::
-   'abrt' will ignore crashes in executables whose absolute path matches
-   one of specified patterns.
-ProcessUnpackaged = 'yes' | 'no'::
-   When set to 'yes', 'abrt' will catch all crashes in the system.
-   When set to 'no', it will catch crashes only in executables which belong
-   to an installed package.
-   The default is 'no'.
+*OpenGPGCheck = 'yes/no'*::
+   If enabled, only crashes in signed packages will be analyzed. The list of
+   trusted public keys used to verify the signature is in /etc/gpg_keys.conf.
+   +
+   Default is 'yes'.
+*BlackList = 'list'*::
+   Crashes in packages listed here will be ignored by ABRT.
+   +
+   Example: nspluginwrapper, valgrind, strace, mono-core, bash
+   +
+   Default is empty.
+*BlackListedPaths = 'list'*::
+   Crashes in paths matching the glob patterns in this list will be ignored
+   by ABRT.
+   +
+   Example: `/usr/share/doc/*, */example*, /usr/bin/nspluginviewer,
+   /usr/lib*/firefox/plugin-container`
+   +
+   Default is empty.
+*ProcessUnpackaged = 'yes/no'*::
+   If enabled, ABRT will catch crashes of all programs in the system (except
+   those suppressed by other settings). If disabled, only crashes in
+   installed packages will be handled.
```

Comment:
I feel like this is overly complicated description.

`If enabled, ABRT will process crashes from unpackaged executables.` ?